### PR TITLE
Changelog view

### PR DESCRIPTION
**Pull request by OmotayoTalabi**
## What does this pull request do?

change log ui that shows all amend referrals changes from pp

## What is the intent behind these changes?

to view all amend referrals changes.

### CODE REVIEW

**Comment by rgforsyth** [comment]
Think this may need a rebase on main?

File: server/models/changelog.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default interface Changelog {
+  changelogId: string
+  referralId: string
```
Comment (kth13) [inline server/models/changelog.ts:3]: will this not capture the old and new value

File: server/routes/amendAReferral/amendAReferralController.ts [hunk 2]
```diff
@@ -132,4 +134,20 @@ export default class AmendAReferralController {
 
     return ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
+
+  async getChangelog(req: Request, res: Response): Promise<void> {
```
Comment (kth13) [inline server/routes/amendAReferral/amendAReferralController.ts:138]: changelog part of logic can be moved to Changelog controller as it is not part of the amend feature

Comment (OmotayoTalabi) [reply server/routes/amendAReferral/amendAReferralController.ts:138]: created the Changelog controller

Comment (kth13) [reply server/routes/amendAReferral/amendAReferralController.ts:138]: But I think you forgot to remove it from here. 😄

**Review by kth13 — APPROVED** [review]
LGTM

**Pull request merged**